### PR TITLE
 "notification-bar"

### PR DESCRIPTION
**Pull request by Mri1662**
Schema Integration for [Section Name] = "notification-bar" #4

### CODE REVIEW

File: layout/theme.liquid [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
         {% section "announcement-bar" %}
         {% section "header" %}
+        {% section "notification-bar" %}
```
Comment (mihir-bombay-studio) [inline layout/theme.liquid:52]: We can dynamically add the section that you made using preset so no need to statically add in the theme file, just remove the changes made in the theme.liquid.

File: sections/notification-bar.liquid [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "Notification Bar",
+  "settings": [
+    {
+      "type": "text",
+      "label": "Notification Text",
+      "id": "notification_text",
+      "default": "This is notification bar"
+    }
+  ]
+}
```
Comment (mihir-bombay-studio) [inline sections/notification-bar.liquid:22]: You can add a preset here so we can dynamically select a section in shopify theme editor
https://shopify.dev/docs/themes/architecture/sections/section-schema#presets

Good read: https://shopify.dev/docs/themes/tools/online-editor

**Review by mihir-bombay-studio — CHANGES_REQUESTED** [review]
@Mri1662 There are couple of small comments I think you can quickly fix it and then we can close this issue 🙌 Thanks!

**Comment by Mri1662** [comment]
Schema Integration for [Section Name] = "notification-bar"  #4 
desktop view
image
Mobile view 
image

**Pull request merged**